frontend/public/index.html

```diff
@@ -164,6 +164,7 @@ <h3>Loading Metabolic Atlas</h3>
     <script>
       document.getElementById("init-loading").style.display = "block";
     </script>
+    <script src="https://unpkg.com/@rdkit/rdkit/dist/RDKit_minimal.js"></script>
```

Comment:
How large is the compressed RDKit? And is it really faster for the user to rely on the CDN rather than Cloudfare when the library is not cached?

Reply:
It is `140KB` minified locally. Not sure how much that is when compressed but I'm happy to try. Would we need to deploy that to `dev` or can you find out on `localhost`?

Reply:
Should be able to work just fine locally given some changes in the Nginx config https://github.com/MetabolicAtlas/MetabolicAtlas/blob/1d20628a87c76b2d740651a015a53fcdc3e112da/nginx/prod.nginx.conf#L96

Reply:
It seems quite tedious to even get this to work locally. According to [the instructions](https://github.com/rdkit/rdkit-js#using-rdkitjs-package-assets), both the `js` and the accompanying `wasm` files need to be included. This would need an additional file loader. I found [this one](https://github.com/BrockReece/vue-wasm), which doesn't seem to be actively developed (might still be fine). The other problem is that the `wasm` file itself is `5MB` locally. I could spend some more time looking into this but it would probably take at least a few more hours and I'm skeptical about having to bundle a `5MB` file to our SPA.

The current solution (CDN script tag) seems to work fine. I think for most cases the performance should be great as well. The script starts loading as soon as the website is loaded and by the time the user gets to a metabolite/compound page it should be ready to go. I have also tested loading a metabolite page directly with uncached CDN (new browser private window) and the image shows up after `2-3s`.

So my recommendation is that we go with the CDN for now, but if you want me to pursue the local option a bit more I can do that as well. Please let me know!

Reply:
I will open this PR up for review now so others can check it out. We can still change how to use the package later if we want to. Should not affect the functionality.